org.opent2t.sample.lamp.superpopular/com.wink.lightbulb/js/thingTranslator.js

```diff
@@ -249,12 +249,16 @@ class Translator {
         return this.postDeviceResource(deviceId, "colourChroma", payload);
     }
 
-    postSubscribe(callbackUrl, verificationRequest) {
-        return winkHub._subscribe(deviceType, deviceId, callbackUrl, verificationRequest);
+    postSubscribe(subscriptionInfo) {
+        subscriptionInfo.deviceId = deviceId;
```

Comment:
We call it deviceId in wink, and controlId in smartthings.  Probably fine standardizing that, but it's internal to the translators and will need to be done across the board (not this PR).

---
In reply to: [92070558](https://github.com/openT2T/translators/pull/136#discussion_r92070558) [](ancestors = 92070558)